src/libs/actions/PersonalDetails.js

```diff
@@ -310,6 +310,49 @@ function openPersonalDetailsPage() {
     API.read('OpenPersonalDetailsPage');
 }
 
+/**
+ * Fetches public profile info about a given user.
+ * The API will only return the accountID, displayName, and avatar for the user
+ * but the profile page will use other info (e.g. contact methods and pronouns) if they are already available in Onyx
+ * @param {Number} accountID
+ */
+function openPublicProfilePage(accountID) {
+    const optimisticData = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
```

Comment:
Adding `isLoading` to `PERSONAL_DETAILS_LIST` changes `personalDetailsList`. Lots of components subscribe to this and rerender causing a performance issue. #37986